src/uu/factor/src/factor.rs

```diff
@@ -225,20 +225,17 @@ pub fn factor(num: u64) -> Factors {
 
 #[cfg(test)]
 mod tests {
-    use super::{factor, Factors};
+    use super::{factor, Decomposition, Exponent, Factors};
```

Comment:
please run rustfmt :)

Reply:
Oops.  I'm giving VSCode a try and didn't realise that it meant I'd need to remember and `fmt` by hand  >_>'
Thanks!